packages/core/src/common/utils/get-ajv-instance.spec.ts

```diff
@@ -116,6 +164,41 @@ describe('getAjvInstance', () => {
           schemaPath: '#/properties/b/type',
         },
       ]);
+
+      const schema6 = {
+        properties: {
+          password: {
+            type: 'string',
+          },
+          confirmPassword: {
+            const: {
+              $data: '1/password',
+            },
+            type: 'string',
+          },
+        },
+        type: 'object',
+      };
+      const data6 = {
+        password: 'superSecretPassword',
+        confirmPassword: 'superSecretPassword'
+      };
+      strictEqual(ajv.validate(schema6, data6), false, 'If $data is false in the configuration, AJV should not be able to validate using $data references.');
+      strictEqual(ajv.errors, [
```

Comment:
The asset function should be `deepStrictEqual` in this case. Otherwise, the function will always throw an error because the arguments are two different arrays (even if their content are similar).